Cesium.Runtime/StdLibFunctions.cs

```diff
@@ -139,13 +155,21 @@ public static int Atoi(byte* ptr)
 
     public static byte* GetEnv(byte* ptr)
     {
-        var str = StdIoFunctions.Unmarshal(ptr);
-        if (str is null)
+        var envKey = StdIoFunctions.Unmarshal(ptr);
+
+        if (envKey is null)
         {
             return null;
         }
 
-        return StdIoFunctions.MarshalStr(Environment.GetEnvironmentVariable(str));
+        _envVarsStorage ??= InitEnvVarsStorage();
```

Comment:
I am a little concerned about the thread safety here. If two threads enter this field, then `_envVarsStorage` will just silently leak.

Let's just make it `Lazy`? What do you think?

Reply:
Good point! Fixed it